styles.css

```diff
@@ -17,48 +20,100 @@
   font: bold 24px/31px pt_serifbold, arial, helvetica;
 }
 .logo__circle {
- left: 0;
- top: 0;
- top: 50%;
- transform: translateY(-50%);
- width: 28px;
- height: 28px;
- position: absolute;
+  left: 0;
+  top: 0;
+  top: 50%;
+  transform: translateY(-50%);
+  width: 28px;
+  height: 28px;
+  position: absolute;
 }
 .logo__circle_color_dark-blue {
- background: #0056EC;
- border-radius: 50%;
+  background: #0056EC;
+  border-radius: 50%;
 }
 .logo__circle_color_blue {
- left: 8px;
- top: 56%;
- background: #C8DBFB;
- border-radius: 50%;
- opacity: .7;
+  left: 8px;
+  top: 56%;
+  background: #C8DBFB;
+  border-radius: 50%;
+  opacity: .7;
 }
 .menu {
   display: flex;
   align-items: center;
 }
 .menu__item {
-  padding: 0 13px;
+  position: relative;
+  margin-right: 26px;
+  font: 18px/21px roboto, helvetica, arial, sans-serif;
+  color: #000;
+  text-decoration: none;
+  transition: opacity .5s width .5s;
+}
+
+.menu__item_active::after {
+  position: absolute;
+  display: block;
+  content: '';
+  width: 100%;
+  height: 3px;
+  background: #0056EC;
+}
+
+.menu__item:hover {
+  opacity: .5;
 }
+
+.menu__item:hover::after {
+  display: block;
+  content: '';
+  width: 100%;
+  height: 3px;
+  background: #0056EC;
+}
+
+.menu__item:hover::after,
+.menu__item_active::after {
+  opacity: 1;
+  width: 100%;
+}
+
 .menu__item:last-child {
-  padding-right: 0;
+  margin-right: 0;
+}
+.main {
+  padding: 0 63% 100px 0;
+  background: top right no-repeat url(img/mainlmg..svg);
+}
+.main__title {
+  margin: 0 0 40px 0;
+  font: 50px/65px pt_serifbold, arial, helvetica, sans-serif;
 }
-.a {
- font-family: Roboto;
+.main__subtitle {
+  margin: 0 0 41px 0;
+  font: 18px/21px roboto, arial, helvetica, sans-serif;
 }
-.main-content__title {
- font-family: PT serif;
- margin-bottom: 40px;
+.main__title-text_highlight {
+  background: #ECF3FF;
 }
-.main-content__subtitle{
-  margin-bottom: 40px;
+.main__button {
+  display: inline-block;
+  font: 14px/16px Roboto, arial, helvetica, sans-serif;
+  color: #fff;
+  background: #0056EC;
+  padding: 14px 16px;
+  text-decoration: none;
+  border-radius: 4px;
+  transition: opacity .5s;
+  animation: change-border 5s infinite;
 }
-.p {
- font-family: Roboto;
+
+.main__button:hover{
+  opacity: .5;
 }
-.button {
- font-family: Roboto;
+
+@keyframes change-border {
+  from {border: 3px #ECF3FF;}
```

Comment:
Codacy found an issue: [Expected "#ECF3FF" to be "#ecf3ff" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/HTML_SNN/pullRequest?prid=6259723)